packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -27,7 +27,10 @@ Markup:
   <legend class="ds-c-label">Historical figures</legend>
   <span class="ds-c-field__hint">Select all that apply</span>
   <input class="ds-c-choice" id="truth-1" type="checkbox" name="historical-figures-1" value="truth" checked>
-  <label for="truth-1">Sojourner Truth</label>
+  <label for="truth-1">
+    Sojourner Truth
+    <span class="ds-c-field__hint">Example hint text</span>
```

Comment:
@sawyerh, I hadn't realized I had added this - is this hint text (or technically requirement text) in the CSS example ok, or should I remove it? The React example does include hint/requirement text to demonstrate how to pass it in via props, but these examples didn't originally include them.